doc/source/abfallwirtschaft_germersheim_de.md

```diff
@@ -0,0 +1,40 @@
+# Abfalltermine Forchheim
+
+Support for Landkreis Germersheim schedules provided by [abfalltermine-forchheim.de](https://www.abfallwirtschaft-germersheim.de/) located in Rhineland-Palatinate, Germany.
```

Comment:
Shouldn't this be [abfallwirtschaft-germersheim.de] instead? Couldn't find any connection between forchheim and germersheim when briefly looking at the websites.

Reply:
you're right copied the file and missed changing this line 

changed it and squashed the commit